app/experimenter/nimbus-ui/src/components/TableHighlights/index.tsx

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { AnalysisData } from "../../lib/visualization/types";
+import {
+  HIGHLIGHTS_METRICS_LIST,
+  METRICS_TIPS,
+  METRIC,
+  BRANCH_COMPARISON,
+  SEGMENT_TIPS,
+  TABLE_LABEL,
+} from "../../lib/visualization/constants";
+import { getTableDisplayType } from "../../lib/visualization/utils";
+import { getExperiment_experimentBySlug_featureConfig } from "../../types/getExperiment";
+import TableVisualizationRow from "../TableVisualizationRow";
+import { ReactComponent as Info } from "../../images/info.svg";
+
+type TableHighlightsProps = {
+  featureConfig: getExperiment_experimentBySlug_featureConfig;
+  results: AnalysisData["overall"];
+};
+
+const TableHighlights = ({
+  featureConfig,
+  results = {},
+}: TableHighlightsProps) => {
+  const highlightMetricsList = [
+    {
+      value: `${featureConfig.slug.replace(/-/g, "_")}_ever_used`,
+      name: `${featureConfig!.name} conversion`,
+      tooltip: METRICS_TIPS.CONVERSION,
+    },
+    ...HIGHLIGHTS_METRICS_LIST,
+  ];
+
+  return (
+    <table data-testid="table-highlights">
+      <tbody>
+        {Object.keys(results).map((branch) => {
+          const userCountMetric =
+            results[branch]["branch_data"][METRIC.USER_COUNT];
+          return (
+            <tr key={branch} className="border-top">
+              <th className="align-middle p-1 p-lg-3" scope="row">
+                <p>{branch}</p>
+                <p className="h6">
+                  {userCountMetric[BRANCH_COMPARISON.ABSOLUTE]["point"]}{" "}
+                  participants ({userCountMetric["percent"]}%)
+                </p>
+              </th>
+              <th className="align-middle p-1 p-lg-3" scope="row">
+                All Users&nbsp;
+                <Info data-tip={SEGMENT_TIPS.ALL_USERS} />
+              </th>
+              <td className="p-1 pb-lg-0 p-lg-3">
+                {highlightMetricsList.map((metric) => {
+                  const metricKey = metric.value;
+                  const displayType = getTableDisplayType(
+                    metricKey,
+                    TABLE_LABEL.HIGHLIGHTS,
+                    results[branch]["is_control"],
+                  );
+                  return (
+                    <TableVisualizationRow
+                      key={metricKey}
+                      metricName={metric.name}
+                      results={results[branch]}
+                      tableLabel={TABLE_LABEL.HIGHLIGHTS}
+                      {...{ metricKey, displayType }}
+                    />
+                  );
+                })}
+              </td>
+            </tr>
+          );
+        })}
+      </tbody>
+    </table>
+  );
+};
+
+export default TableHighlights;
```

Comment:
This works for me. I wouldn't use those badges though for `control` and `treatment` because we use badges elsewhere in the design to show `primary metric` `guardrail metric` etc.

Reply:
Oh man @jodyheavener coming in here with FE nitpicks 🤣 FWIW I did think we'd address these later in some FE polish ticket after design review, but I don't mind fixing them here. Thanks for the detailed review BTW!

> A little more space above the table would give it some breathing room (maybe my-4)
> Wrapping "All Users" in a span and giving it align-middle will help it vertically line up with the SVG

👍

> There appears to be a bottom margin on the last paragraph in each column, resulting in some funky extra space. If we remove that it looks a little more vertically-aligned

Huh I actually meant to remove that padding-bottom with `p-1 pb-lg-0 p-lg-3` but it doesn't work, this is why I don't like `!important`, `padding-bottom` should be a more specific property 🤦‍♀️ Changed to `pt-1 pl-1 pr-1 pt-lg-3 pr-lg-3 pl-lg-3` which looks nuts but looks nice. If you have a better class name set let me know, heh.

> It would be nice to have a border-bottom on the last row

I left this out since the Overview table is right below it and figured it would have a top/bottom border.

Per Marina's comment I'll leave out the pills.

> This could also benefit from a light background to give it some depth - I used `bg-light`

I actually think this looks nicer with the white background, otherwise it looks like it should have left/right borders. Maybe someone else will disagree. Actually, according to Figma, it looks like the entire page should have a light grey BG. Maybe users will like all the tables to be light grey though, I am not sure, let's table that for now? (see what I did there)

Reply:
Thanks for updating this. I would still say that the page could benefit from some distinction between the table and the rest of the page, and unless we are also removing the margin between this table and the Overview table below it should prob still have its own bottom border. But both of these can be addressed later.